express/parsers/apps/espresso/parser.py

```diff
@@ -346,6 +347,10 @@ def final_structure_strings(self) -> List[str]:
 
 class EspressoParser(EspressoLegacyParser):
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.xml_data_file = "data-file-schema.xml"
```

Comment:
`xml_data_file_name`

Reply:
Fixed, and refactored the `find_xml_file` method above for readability